### ISSUE TITLE
Get-DbaAgentJobOutputFile - opening output files does not work

### ISSUE THREAD

**Issue by MikeyBronowski** [sqlcollaborative/dbatools#7684]
### Verified issue does not already exist?

Yes

### What error did you receive?

The two examples won't work as 
1) the OpenFile parameter does not exist
2) FileName is not a proper name of the property
3) When the tokens are used the path is not translated into an actual file path, i.e. the output will have this 
`\\localhost\$(ESCAPE_SQUOTE(SQLLOGDIR))\$(ESCAPE_SQUOTE(JOBNAME))_$(ESCAPE_SQUOTE(STEPID))_$(ESCAPE_SQUOTE(DATE))_$(ESCAPE_SQUOTE(TIME)).txt`

```
    .EXAMPLE
        $Servers = 'SERVER','SERVER\INSTANCE1'
        Get-DbaAgentJobOutputFile -SqlInstance $Servers -Job 'The Agent Job' -OpenFile

        This will return the configured paths to the output files for each of the job step of the The Agent Job Job
        on the SERVER instance and the SERVER\INSTANCE1 and open the files if they are available

    .EXAMPLE
        PS C:\> (Get-DbaAgentJobOutputFile -SqlInstance SERVERNAME | Out-GridView -PassThru).FileName | Invoke-Item

        This will return the configured paths to the output files for each of the job step of all the Agent Jobs
        on the SERVERNAME instance and Pipe them to Out-GridView and enable you to choose the output
        file and open it
```

### Steps to Reproduce

Run the examples above.

### Are you running the latest release?

Yes

### Other details or mentions

Probably those examples can be removed at this point.

Paging @SQLDBAWithABeard who authored this command.

### What PowerShell host was used when producing this error

Windows PowerShell (powershell.exe), Windows PowerShell ISE (powershell_ise.exe)

### PowerShell Host Version

Name                           Value                                                                                   
----                           -----                                                                                   
PSVersion                      5.1.19041.1023                                                                          
PSEdition                      Desktop                                                                                 
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                 
BuildVersion                   10.0.19041.1023                                                                         
CLRVersion                     4.0.30319.42000                                                                         
WSManStackVersion              3.0                                                                                     
PSRemotingProtocolVersion      2.3                                                                                     
SerializationVersion           1.1.0.1                                                                                 

### SQL Server Edition and Build number

Microsoft SQL Server 2017 (RTM-CU13) (KB4466404) - 14.0.3048.4 (X64)   Nov 30 2018 12:57:58   Copyright (C) 2017 Microsoft Corporation  Developer Edition (64-bit) on Linux (Ubuntu 16.04.5 LTS)

**Comment by niphlod** [comment]
ahum, funny. 
we have the whole logic in Get-DbaAgentJobHistory for the tokens, either we copy it or we refactor.
But the probl is that tokens are only valuable when seeing the history . I think this just makes a pair with Set-DbaAgentOutputFile, and is good only for fixed "append-only" paths, of course.
-OpenFile is not a param, so that example doesn't make sense.
The other one just for fixed paths so............ yeah, sure, I think the proper thing to do here is:
- drop those examples 
- replace @niph**old** with @niph**lod** ;-)

**Comment by MikeyBronowski** [comment]
@niphlod oh, i did not notice you there - apologies. Well, technically not you ;-)

I will raise a PR removing the examples and fixing the "old'.

**Comment by MikeyBronowski** [comment]
Fixed in #7683
